common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -200,6 +200,15 @@ class CapaFields(object):
         scope=Scope.settings
     )
 
+    @property
+    def non_editable_metadata_fields(self):
```

Comment:
@cpennington: I just tried that, and it doesn't work. I suspect that one of the reasons it doesn't work is that each of these classes defines their own `data` field, rather than relying on inheritance. (I don't even know if `EditingDescriptor` is in the inheritance chain for all of these classes.)